ksqldb-functional-tests/src/test/resources/qtt_test_cases/incorrect/value_schema_mismatch/8.0.0_1735879593796/topology

```diff
@@ -6,11 +6,14 @@ Topologies:
       --> WhereFilter
       <-- KSTREAM-SOURCE-0000000000
     Processor: WhereFilter (stores: [])
-      --> Project
+      --> KSTREAM-PEEK-0000000003, Project
```

Comment:
If a query does a repartition and a different sub topology is created, would this behaviour cause renaming in subsequent state stores?

Reply:
No. Given the behaviour of naming is restored for the function, it won't cause state store renaming.